autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -1,81 +1,108 @@
 import asyncio
-import time
+#import time
+import datetime
 from typing import List
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
 from scraper.models.course import generate_course_id
-from scraper.models.section import generate_meeting_id, generate_meeting_time
+from scraper.models.section import generate_meeting_id
 
-def parse_section(courseData,instructor_object):
+def generate_meeting_time(string_time: str):
+    """ Generates a meeting time from a string in format hhmm.
+            ex) 1245 = 12:45am. 1830 = 6:30 pm."""
```

Comment:
I think `convert_meeting_time` would be a more applicable name, as the function is given a string and _converts_ it to a `datetime.time` object.

Reply:
That makes sense. Because there is no new data being created, I can see why generate instead of convert could be misleading. Will change.